packages/camp/package.py

```diff
@@ -4,6 +4,7 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 import os
+    depends_on("blt@0.6.2:", type="build", when="@2024.02.1:")
```

Comment:
I'm gonna guess this isnt right =)

Reply:
I need automated testing in that repo, my copy-paste skills are dubious...